.github/templates/main.yml

```diff
@@ -228,4 +248,23 @@ jobs:
           path: wrappers/build
       - #@ template.replace(buildPackages())
       - #@ findPackageVersion()
-      - #@ template.replace(uploadPackageArtifacts())
\ No newline at end of file
+      - #@ template.replace(uploadPackageArtifacts())
+  run-tests-net-framework:
+    runs-on: windows-latest
+    name: Run tests for NET framework
+    needs: build-packages
+    steps:
+      - name: Add msbuild to PATH
+        uses: microsoft/setup-msbuild@v1.0.2
+      - #@ checkoutCode()
+      - #@ template.replace(fetchPackageArtifacts( [ "Realm", "Realm.Fody" ] ))
+      - #@ buildNetFrameworkTest("Tests/Realm.Tests", TargetFramework="net461" )
+      - name: Run the tests
+        run: #@ "${{ github.workspace }}/Tests/Realm.Tests/bin/" + configuration + "/net461/Realm.Tests.exe --result=TestResults.Windows.xml --labels=After"
+      - name: Publish Unit Test Results
+        uses: EnricoMi/publish-unit-test-result-action/composite@v1
+        if: always()
+        with:
+          files: TestResults.Windows.xml
+          comment_on_pr: false
+          check_name: Windows .NET framework tests results
```

Comment:
Is this something we'll need in the future? Might be worth extracting it to a function, but might as well just do it in a follow up PR if we need to.

Reply:
It's already done in the other PR, which is a branch of this. You can see it [here](https://github.com/realm/realm-dotnet/blob/4457127e51a7baf7f16945561290ed5184de5f88/.github/templates/main.yml#L107).
I didn't merge it backward.